### ISSUE TITLE
UX Improvement to resizing a selection of frames 

### ISSUE THREAD

**Issue by richiewhelan** [excalidraw/excalidraw#6793]
Action: I want to resize a group of frames and all of the contents inside of those frames

- When I click and drag to select a large group of frames
- The anticipated behaviour is that resizing this selection will resize all frames and all contents within frames - in a similar way to how groups or regular selection works
- Right now, only the frames resize and the internal contents get left behind at their original size:

<img width="778" alt="Screenshot 2023-07-21 at 15 26 38" src="https://github.com/excalidraw/excalidraw/assets/5108043/c758b206-f4e5-4419-aa18-f4b37cc76159">

<img width="1123" alt="Screenshot 2023-07-21 at 15 29 57" src="https://github.com/excalidraw/excalidraw/assets/5108043/977269de-1bcd-4b7f-a1f0-4d4d9cfd52bb">

**Comment by ad1992** [comment]
I agree we should be resize the frame contents as well when multiple frames resizing. 
Tldraw, miro does similarly as well.

cc @ryan-di

**Comment by 7flash** [comment]
Currently frame resizing has inconsistent behavior. If I resize from left top  corner, it moves coordinates of all elements around the canvas as well, but if I resize from bottom right corner,  elements position stays the same.

**Comment by dwelle** [comment]
agreed. We'll fix.

> Currently frame resizing has inconsistent behavior. If I resize from left top corner, it moves coordinates of all elements around the canvas as well, but if I resize from bottom right corner, elements position stays the same.

It's not inconsistent, but I agree it's not ideal. More info https://github.com/excalidraw/excalidraw/issues/7233
